Makefile

```diff
@@ -54,23 +54,26 @@ version: ## output to version
 # Docker local development tasks
 ## Dargo == Docker Cargo
 dargo-prep: ## This runs everything neccessary to start developing locally in a container
-	# Spin up a long-running rust container
+	@# Spin up a long-running rust container
 	make dargo-create-container
 	# Compile and cache all dependencies
-	make dargo COMMAND=build
-	# Perform a first-sweep check, to fill up the cache (it seems to be different than the build one)
-	make dargo COMMAND=check
-    # Compile and cache test dependencies, run tests
-	make dargo COMMAND=test
+	@make dargo COMMAND=build
+	@# Perform a first-sweep check, to fill up the cache (it seems to be different than the build one)
+	@make dargo COMMAND=check
+	@# Compile and cache test dependencies, run tests
+	@make dargo COMMAND=test
 	# Now all caches are filled, and subsequent operations will be faster
 	# Consider adding dargo(){ make dargo COMMAND=$1} to your ~/.zshrc or ~/.bashrc for ergonomics
 
 dargo: ## Run a cargo command inside the container
+	@# If the dargo container does not exist, create it
+	@if [ -z $(shell docker ps | grep "$(LOCAL_DEV_CONTAINER_NAME)" | cut -c1-12) ]; then make dargo-prep; fi
```

Comment:
You can simplify these pipes to a single docker ps call: `docker ps --format "{{.ID}}" --filter "name=$(LOCAL_DEV_CONTAINER_NAME"`

Reply:
Docker has go template-style formatting to the output? Cool! And also filtering? Dope! :)